react/getting_started_with_react/passing_data_between_components.md

```diff
@@ -63,7 +63,7 @@ export default function App() {
 }
 ```
 
-This may not seem like a huge deal right now, but what if we had 10 buttons, each one having different text, fonts, colors, sizes, and any other variation you can think of. Creating a new component for each of these button variations would very quickly lead to a LOT of code duplication.
+This may not seem like a huge deal right now, but what if we had 10 buttons, each one having different text, fonts, colors, sizes, and any other variation you can think of? Creating a new component for each of these button variations would very quickly lead to a LOT of code duplication.
```

Comment:
I don't think this is a question, so I'm not sure this makes it any more grammatically correct.

Reply:
Since it starts with "what if", it seems natural for it to end with a question mark.

Reply:
In that case we probably want the question to be something like:

> What if we had ten buttons?

Reply:
That sounds good. I'll make the change.